Makefile

```diff
@@ -21,7 +21,7 @@ else
 ARCHDIRS=$(ARCH)_$(BITSIZE) $(ARCH)
 endif
 
-DIRS=lib common $(ARCHDIRS) backend cfrontend driver debug\
+DIRS=lib common $(ARCHDIRS) backend cfrontend driver \
   flocq/Core flocq/Prop flocq/Calc flocq/Appli exportclight \
```

Comment:
To explain this change: there are no .v and .vo files in directory debug/, so having it in DIRS is not necessary, and causes an installation error because `debug/*.vo` doesn't exist.